frontend/src/components/OrderSummary.tsx

```diff
@@ -45,15 +48,20 @@ const OrderSummary = ({ restaurant, cartItems, removeFromCart }: Props) => {
                 className="cursor-pointer"
                 onClick={() => removeFromCart(item._id)}
               />
-              ${item.price * item.quantity}
+              ${((item.price * item.quantity) / 100).toFixed(2)}
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/omkardongre/Food-Ordering-App/pull/47#issuecomment-2436111848>#96fecf</a><span> - 10/24/2024, 07:07 pm</span>
<br>

<div><small>🟠</small> Medium importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> Consider handling potential floating-point precision issues when performing calculations with monetary values. The current implementation might lead to rounding errors.
</div>
<div id="fix">
<b>Fix:</b> To address potential floating-point precision issues, consider using Math.round() before division in your calculations. For example:
${(Math.round(item.price * item.quantity) / 100).toFixed(2)}
This approach helps minimize rounding errors without introducing external dependencies. If more precise calculations are needed, consider using a library like 'decimal.js'.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -51 +51 @@
 -              ${((item.price * item.quantity) / 100).toFixed(2)}
 +              ${(Math.round(item.price * item.quantity) / 100).toFixed(2)}
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged